### PR TITLE
change workspace name and delete workspace ui

### PR DESCRIPTION
**Pull request by codesmiles**

### CODE REVIEW

File: control/src/pages/settings/components/changeWorkspaceName.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import "../styles/changeWoekspaceName.module.css";
```
Comment (ickynavigator) [inline control/src/pages/settings/components/changeWorkspaceName.js:2]: import the styles properly and check this file name

File: control/src/pages/settings/components/deleteWorkspace.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
+import "bootstrap/dist/css/bootstrap.min.css";
+import '../styles/deleteWorkspace.module.css';
```
Comment (ickynavigator) [inline control/src/pages/settings/components/deleteWorkspace.js:2]: same problem here

File: control/src/pages/settings/styles/changeWorkspaceName.module.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+*{
```
Comment (ickynavigator) [inline control/src/pages/settings/styles/changeWorkspaceName.module.css:3]: don't use the universal selector

File: control/src/pages/settings/styles/deleteWorkspace.module.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+*{
```
Comment (ickynavigator) [inline control/src/pages/settings/styles/deleteWorkspace.module.css:3]: don't use the universal selector

**Review by ickynavigator — CHANGES_REQUESTED** [review]

File: control/src/pages/settings/styles/deleteWorkspace.module.css (hunk 4 above)
Comment (codesmiles) [reply control/src/pages/settings/styles/deleteWorkspace.module.css:3]: Ok noted

**Pull request closed without merging**